README.md

```diff
@@ -114,14 +114,17 @@ There are several approaches of addressing the pile-up in the detector:
   It creates histograms filling in the information on the deposits per event. This can be later scaled with *sqrt(mu)* for the noise (RMS of the energy distributions) and with *mu* for the mean energy deposit.
   Detailed analysis and this scaling is done with FCC_calo_analysis_cpp toolkit.
 
-- apply estimated noise levels at the cluster level for sliding window reconstruction or layer by layer for the topological clusters (**--addPileupNoise**).
+- apply estimated noise levels at the cluster level for sliding window reconstruction or per cell for the topological clusters (**--addPileupNoise**).
```

Comment:
Hi Anna, the pileup noise is added per cell dependent on its layer and eta, without any corrections due to correlations.

Reply:
So the results you are showing do not take into account any correlations?
What about the study you did where you tried to parametrise the pileup noise for each layer as a function of the number of cells used from this layer (to build a cluster) ? This would at least take into account the correlations within one layer.